assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -117,6 +117,30 @@ const DeckLayoutFooter: React.FC = () => {
   const [nextButtonText, setNextButtonText] = useState('Next');
   const [nextCheckButtonText, setNextCheckButtonText] = useState('Next');
 
+  const handleValueExpression = (operationValue: string) => {
```

Comment:
lets separate this function out and export it at the top (passing in the currentActivityTree) so that we can unit test it with various types of values. I worry about the variable parsing.